pkg/daemon/update.go

```diff
@@ -556,6 +572,15 @@ func (dn *Daemon) updateKernelArguments(oldConfig, newConfig *mcfgv1.MachineConf
 	return exec.Command("rpm-ostree", args...).Run()
 }
 
+// updateToRealTimeKernel updates to RT Kernel if it is set to true.
+// In case realtime is set to false, switch to traditional Kernel if already
+// on RT kernel, otherwise do nothing
+func (dn *Daemon) updateToRealTimeKernel(oldConfig, newConfig *mcfgv1.MachineConfig) error {
+	// TODO
```

Comment:
See https://github.com/williamcaban/ocp-rt/blob/master/rhcos-rt.sh#L18

This will also need to reconcile and handle the case where we're upgrading from an older RT kernel; see https://github.com/williamcaban/ocp-rt/blob/master/README.md#swap-in-an-updated-rt-kernel for that.

Reply:
right, thanks for the link

Reply:
In that case, do we want to be more specific and have the RT kernel version specified in the MC?

Reply:
I don't think we need to support that; haven't heard of any asks related to having multiple RT kernel choices.  We're only embedding one version of the RT kernel in the oscontainer today (same as we only have one version of the mainline kernel).

Anyone who wants to e.g. test something out of band is probably best using direct `rpm-ostree override` commands via e.g. `oc debug node/`.